--- src/Makefile.am.orig	2008-11-13 17:30:57.000000000 -0500
+++ src/Makefile.am	2008-11-13 17:31:42.000000000 -0500
@@ -47,5 +47,8 @@
   SVGCanvas.cpp SVGCanvasItem.cpp NodeList.cpp $(FFMPEG_SRC)
 
 libwxsvg_la_LIBADD = svgxml/libwxsvgxml.la $(RENDER_LIB) $(TEXT_LIB)
+libwxsvg_la_CFLAGS = $(LIBAVFORMAT_CFLAGS)
+libwxsvg_la_LDFLAGS = $(LIBAVFORMAT_LDFLAGS)
+
 
 INCLUDES = -I$(top_builddir)/include -I$(top_builddir)/include/wxSVG
