--- src/svgxml/Makefile.am.orig	2008-11-13 21:33:02.000000000 -0500
+++ src/svgxml/Makefile.am	2008-11-13 21:36:12.000000000 -0500
@@ -1,7 +1,5 @@
-SUBDIRS = expat
-
 noinst_LTLIBRARIES = libwxsvgxml.la
 libwxsvgxml_la_SOURCES = svgxml.cpp svgxmlhelpr.cpp
-libwxsvgxml_la_LIBADD = expat/libexpat.la
+libwxsvgxml_la_LIBADD = $(EXPAT_LIBS)
 
 INCLUDES = -I$(top_builddir)/include/wxSVGXML
--- configure.in.orig	2008-12-11 13:35:19.000000000 -0500
+++ configure.in	2008-12-11 13:35:24.000000000 -0500
@@ -179,7 +179,6 @@
   include/wxSVGXML/Makefile
   include/wxSVG/Makefile
   src/svgxml/Makefile
-  src/svgxml/expat/Makefile
   src/Makefile
   src/agg/Makefile
   src/cairo/Makefile
--- src/svgxml/svgxml.cpp.orig	2008-12-11 13:39:49.000000000 -0500
+++ src/svgxml/svgxml.cpp	2008-12-11 13:40:00.000000000 -0500
@@ -21,11 +21,7 @@
 
 #include "svgxml.h"
 
-#ifdef SYSEXPAT
 #include <expat.h>
-#else
-#include "expat/xmlparse.h"
-#endif
 
 #include "wx/wfstream.h"
 #include "wx/datstrm.h"
